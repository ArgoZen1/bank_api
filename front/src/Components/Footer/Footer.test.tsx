import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer component', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should display the correct text', () => {
        const { getByText } = render(<Footer />);
        const text = getByText(/Copyright 2020 Argent Bank/i);
        expect(text).toBeTruthy();
    });
});