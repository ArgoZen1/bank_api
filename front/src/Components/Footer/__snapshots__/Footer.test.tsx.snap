// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      class="footer-text"
    >
      Copyright 2020 Argent Bank
    </p>
  </footer>
</DocumentFragment>
`;
